packages/indexer/src/transaction/serializer.ts

```diff
@@ -354,6 +531,18 @@ export async function getTransaction(
   }
 }
 
+export function parseFormattedInstructionArgsData<T>(data: string) {
+  let jsonString = data.replace(/'/g, '"');
+
+  jsonString = jsonString.replace(/(\w+):/g, '"$1":');
```

Comment:
I'd be careful with this. Given not everything is always the same or ordered the same, so it's just something to be considerate of.